### PR TITLE
fix: use our brand font on mobile

### PR DESCRIPTION
**Pull request by y-lohse**
Slightly surprised this is all it took:

- added font-face instructions on mobile
- added the font files
- configured webpack's `file-loader` to handle font files

### CODE REVIEW

**Comment by y-lohse** [comment]
[🐛📱 Corriger la police de caractère qui n'est plus Lato sur iOS](https://trello.com/c/SZXkiiFn/658-%F0%9F%90%9B%F0%9F%93%B1-corriger-la-police-de-caract%C3%A8re-qui-nest-plus-lato-sur-ios)

File: config/webpack.target.mobile.js [hunk 1]
```diff
@@ -15,6 +15,17 @@ module.exports = function(production, app) {
       path: path.resolve(__dirname, `../targets/${app}/mobile/www`),
       filename: '[name].js'
     },
+    module: {
+      rules: [
+        {
+            test: /\.(eot|svg|ttf|woff|woff2)$/,
```
Comment (enguerran) [inline config/webpack.target.mobile.js]: Why did you add `svg` file?

Comment (enguerran) [reply config/webpack.target.mobile.js]: Same question for `ttf`, `eot` ;)

Comment (y-lohse) [reply config/webpack.target.mobile.js]: I felt like it would be a good idea to cover all possible font formats, but I think I should at least remove svg now that you say it. We use it for loads of different things, better not mix up the loaders.

Comment (enguerran) [reply config/webpack.target.mobile.js]: > Don’t write code in anticipation of features that haven’t been requested. You aren’t going to need it (YAGNI).
